File: Backend/demo/src/main/java/com/example/Models/UserLogin.java
package com.example.Models;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity
public class UserLogin {
    
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long userid;
    
    private String userName;
    
    private String firstName;
    
    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    private String lastName;
    
    private String email;
    
    private String password;
    
    private String role;

    public Long getUserid() {
        return userid;
    }

    public void setUserid(Long userid) {
      
        this.userid = userid;
    }

    public String getUserName() {
    
        return userName;
    }

    public void setUserName(String userName) {
     
        this.userName = userName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
      
        this.lastName = lastName;
    }

    public String getEmail() {
     
        return email;
    }

    public void setEmail(String email) {
      
        this.email = email;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
      
        this.password = password;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
       
        this.role = role;
    }

    public UserLogin() {
    }

    public UserLogin(Long userid, String userName, String firstName, String lastName, String email, String password, String role) {
        super();
        this.userid = userid;
        this.userName = userName;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.password = password;
        this.role = role;
        
    }

    @Override
    public String toString() {
        return "UserDetails [userid=" + userid + ", userName=" + userName + ", firstName=" + firstName + ", lastName="
                + lastName + ", email=" + email + ", password=" + password + ", role=" + role + "]";
    }
}

File: Dot net Backend/latest/Fleet_Management/Models/UserLogin.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace Fleet_Management.Models;

[Table("user_login")]
public partial class UserLogin
{
    [Key]
    [Column("userid")]
    public long Userid { get; set; }

    [Column("email")]
    [StringLength(255)]
    public string? Email { get; set; }

    [Column("first_name")]
    [StringLength(255)]
    public string? FirstName { get; set; }

    [Column("last_name")]
    [StringLength(255)]
    public string? LastName { get; set; }

    [Column("password")]
    [StringLength(255)]
    public string? Password { get; set; }

    [Column("role")]
    [StringLength(255)]
    public string? Role { get; set; }

    [Column("user_name")]
    [StringLength(255)]
    public string? UserName { get; set; }
}
